src/Module/Admin/DBSync.php

```diff
@@ -102,13 +102,13 @@ protected function content(array $request = []): string
 
 				if (!count($failed)) {
 					$o = Renderer::replaceMacros(Renderer::getMarkupTemplate('admin/dbsync/structure_check.tpl'), [
-						'$base' => DI::baseUrl()->get(true),
+						'$base' => DI::baseUrl(),
 						'$banner' => DI::l10n()->t('No failed updates.'),
 						'$check' => DI::l10n()->t('Check database structure'),
 					]);
 				} else {
 					$o = Renderer::replaceMacros(Renderer::getMarkupTemplate('admin/dbsync/failed_updates.tpl'), [
-						'$base' => DI::baseUrl()->get(true),
+						'$base' => DI::baseUrl(),
```

Comment:
Same remark as for the `'$baseurl'` template variable, except it should be renamed in `admin/dbsync/failed_updates.tpl`
```suggestion
```